### ISSUE TITLE
Some place to discuss/talk

### ISSUE THREAD

**Issue by Baklap4** [OkGoDoIt/OpenAI-API-dotnet#73]
Currently there's no place to discuss certain things except for the issue tracker.
Personally I love to use Discord for this as opening a lot of issues as questions just litters the project in my honest opinion.
Also the discussion tab from Github is a place which is missed a lot so therefore creating some place of realtime messaging is handy.

Let me know your thoughts and if you need help with it 👍

**Comment by Bia10** [comment]
I completely agree, there's a lot to be done to improve this project especially making it consistently and coherently adhere to fundamental design principles. A place to discuss is must. I also vote for discord.

**Comment by ernop** [comment]
Agree. I would like to discuss how we're going to quickly enable GPT4 once it shows up in the API backend endpoints.

**Comment by Baklap4** [comment]
@OkGoDoIt If you're alright with it (please let us know) i can create a discord server and configure it so it can be used properly :)

**Comment by Baklap4** [comment]
Added a place to discuss, but i can't add features like github issues/pr/release sync as i need a webhook for that.
For now the discord server is online at: https://discord.gg/ysrM6UF9gC
A PR will follow to include a link to this server from the readme
